### PR TITLE
PollingStation property alignment with EML/Figma

### LINKED ISSUE #684: Polling station address: Align backend with design
Align fields in the database with EML design and frontend.

- address
- house_number
- house_number_addition

The backend currently still has separate fields that should be condensed into a single field.

### PR DESCRIPTION
**Pull request by lkleuver**
This changes the backend PollingStation model to reflect the EML and Figma design files
removing street, house_number and house_number_addition replacing it with "address"

PollingStationType has also been made optional.

Concerns:
PDF rendering since properties were removed

closes #684

### CODE REVIEW

File: backend/templates/inputs/model-na-31-2.json [hunk 1]
```diff
@@ -633,9 +549,7 @@
       "number": 10,
       "number_of_voters": 123,
       "polling_station_type": "FixedLocation",
-      "street": "Teststraat van de lange staatnaam der straten van de straatnamen",
-      "house_number": 1,
-      "house_number_addition": null,
+      "address": "Teststraat van de lange staatnaam der straten van de straatnamen",
```
Comment (Lionqueen94) [inline backend/templates/inputs/model-na-31-2.json:552]:
```suggestion
      "address": "Teststraat van de lange straatnaam der straten van de straatnamen",
```

**Review by praseodym — CHANGES_REQUESTED** [review]
CI fails because the frontend API client hasn't been updated yet

**Review by Lionqueen94 — DISMISSED** [review]

**Review by praseodym — APPROVED** [review]

File: frontend/scripts/openapi/generator.ts [hunk 2]
```diff
@@ -135,6 +135,19 @@ function tsType(s: ReferenceObject | SchemaObject | undefined): string {
...
+        ?.map((obj) => {
+          if ("$ref" in obj) {
+            return obj.$ref.substring(obj.$ref.lastIndexOf("/") + 1);
+          }
+          return obj.type || "unknown";
+        })
+        .join(" | ") ?? "unknown"
+    );
+  }
+
```
Comment (Lionqueen94) [inline frontend/scripts/openapi/generator.ts:150]: Is this still needed now?

Comment (praseodym) [reply frontend/scripts/openapi/generator.ts:150]: I don't think we have to remove this, it's fine if the generator supports a bit more than we need.

**Review by praseodym — APPROVED** [review]

**Review by Lionqueen94 — APPROVED** [review]

**Pull request merged**